### ISSUE TITLE
Refactor Clipboard handling

### ISSUE THREAD

**Issue by micieslak** [status-im/status-desktop#16196]
## Description

Currently there are two ways of handling system's `clipboard`:
1. `QClipboardProxy` - C++ wrapper exposed to QML as a singleton
2. `NIM`'s `copyToClipboard` method

The goal is to remove `NIM`'s part and consistently use refactored version of `QClipboardProxy`.

Current, backend-dependent clipboard handling is a big obstacle on the way toward non-singleton stores and stores exposing given functionality only once (as described in contribution guide).

There are some steps to do:

1. Remove `copyToClipboard` from `utils.nim` and other places in nim code
2. Remove clipboard support in `DOtherSide` as it's no longer needed (`dos_qguiapplication_clipboard_setText` and others).
3. Remove
    ```qml
        function copyToClipboard(textToCopy) {
            globalUtils.copyToClipboard(textToCopy)
        }
    ```
    which is copy-pasted in 10+ places in QML code (!)
4. Rename `QClipboardProxy` to e.g. `ClipboardUtils`
5. Move `QClipboardProxy::isValidImageUrl` and `QClipboardProxy::getFileSize` to more adequate locations as they are not related to `clipboard`

**Comment by caybro** [comment]
Ad 5: I'd move them to sth like "FileUtils"
